libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -214,22 +214,22 @@ class AppHeaderCore extends Component<IAppHeaderProps & WrappedComponentProps, I
     });
 
     private getTrialCountdown = (expiredDate: string) => {
-        const trialDaysLeft = differenceInCalendarDays(new Date(expiredDate), new Date()) + 1; // include expired date.
-        if (trialDaysLeft === 1) {
+        const trialDaysLeft = differenceInCalendarDays(new Date(expiredDate), new Date()) - 1; // subtract expired date
+        if (trialDaysLeft === 0) {
             return <FormattedMessage id="gs.header.countdown.lastDay" />;
         }
         if (trialDaysLeft === 30) {
             return <FormattedMessage id="gs.header.countdown.oneMonthLeft" />;
         }
-        if (trialDaysLeft > 1 && trialDaysLeft < 30) {
+        if (trialDaysLeft > 0 && trialDaysLeft < 30) {
             return (
                 <FormattedMessage
                     id="gs.header.countdown.numberOfDaysLeft"
```

Comment:
If `trialDaysLeft` is 1, I think we should create a new translation for this case, otherwise the text would be `1 days left` (`days` in plural for this case is grammatically incorrect, isn't it?)

To be consistent with the translation to deal with the case when there is 1 month left at https://github.com/gooddata/gooddata-ui-sdk/blob/e08f121214b492a7e2cb89a9c8fc2040f409ba56/libs/sdk-ui/src/base/localization/bundles/en-US.json#L384